modules/axtask/src/run_queue.rs

```diff
@@ -521,9 +554,11 @@ impl AxRunQueue {
             let prev_ctx_ptr = prev_task.ctx_mut_ptr();
             let next_ctx_ptr = next_task.ctx_mut_ptr();
 
-            // Store the weak pointer of **prev_task** in **next_task**'s struct.
+            // Store the weak pointer of **prev_task** in percpu variable `PREV_TASK`.
             #[cfg(feature = "smp")]
-            next_task.set_prev_task(prev_task.as_task_ref());
+            {
```

Comment:
Remove `{}`

Reply:
`PREV_TASK` is a global percpu static variable decorated by percpu macros, remove this `{}` may cause some grammer problems?